### PR TITLE
Feat: View change logs of invoices

### PR DESCRIPTION
**Pull request by apoorv1316**
Notion: https://www.notion.so/saeloun/Backend-Admin-owner-user-should-be-able-to-view-the-change-log-for-an-invoice-844e71c756be4ffca24e192d65a7e039

What
- Added backend code to view the changelog of invoice events (create, view, send, update, destroy and download)

Preview
Screenshot 2023-06-06 at 12 35 31 PM

### CODE REVIEW

**Review by gowsik-ragunath — CHANGES_REQUESTED** [review]
Added few minor comments

**Comment by apoorv1316** [comment]
@gowsik-ragunath Committed your suggestions. Please review

File: config/routes.rb [hunk 1]
```diff
@@ -22,13 +22,13 @@ def draw(routes_name)
...
-    ActiveSupport::SecurityUtils.secure_compare(
-      ::Digest::SHA256.hexdigest(username),
-      ::Digest::SHA256.hexdigest(ENV["SIDEKIQ_USERNAME"])) &
       ActiveSupport::SecurityUtils.secure_compare(
-        ::Digest::SHA256.hexdigest(password),
-        ::Digest::SHA256.hexdigest(ENV["SIDEKIQ_PASSWORD"]))
-  end
+        ::Digest::SHA256.hexdigest(username),
+        ::Digest::SHA256.hexdigest(ENV["SIDEKIQ_USERNAME"])) &
+        ActiveSupport::SecurityUtils.secure_compare(
```
Comment (prasanthchaduvula) [inline config/routes.rb:28]: this indenetation changes is wrong and no code changes in this.

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Review by prasanthchaduvula — APPROVED** [review]

**Review by gowsik-ragunath — APPROVED** [review]
LGTM

**Pull request merged**